src/components/keypad.js

```diff
@@ -121,7 +122,7 @@ class Keypad extends React.Component {
         };
 
         return <View style={style}>
-            {children}
+            {active && children}
```

Comment:
Can you explain this change for me please?

Reply:
I added this so that the individual buttons only show up in the DOM if the keyboard has been activated. It was doing a thing where even when the keyboard wasn't on the page, all the buttons were showing up in the form controls rotor in the screen reader.

I'll add a comment in the code for clarity!